### PR TITLE
feat: [PIE-3076]: adding schema description for dto variables for environment group

### PR DESCRIPTION
**Pull request by prashantsharma04**
You can use the following comments to re-trigger PR Checks

- Compile: `trigger compile`
- runAeriformCheck: `trigger AeriformCheck`
- CodeFormat: `trigger codeformat`
- MessageMetadata: `trigger messagecheck`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- runDockerizationCheck: `trigger dockerizationcheck`
- runAuthorCheck: `trigger authorcheck`
- Checkstyle: `trigger checkstyle`
- PMD: `trigger pmd`
- TI-bootstrap: `trigger ti0`
- TI-bootstrap1: `trigger ti1`
- TI-bootstrap2: `trigger ti2`
- TI-bootstrap3: `trigger ti3`
- TI-bootstrap4: `trigger ti4`
- FunctionalTest1: `trigger ft1`
- FunctionalTest2: `trigger ft2`
- CodeBaseHash: `trigger codebasehash`

---
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/31423)

### CODE REVIEW

**Review by namanvermah — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/31423)* status: 4 file(s) to review. 1 change requests. 0 of 2 approvals obtained. 

___
*[960-ng-core-beans/src/main/java/io/harness/ng/core/envGroup/constant/EnvironmentGroupConstants.java, line 7 at r1](https://reviewable.io/reviews/harness/harness-core/31423#-MyBAgM4Br8yF-EmEMso:-MyBAgM5-dOZdyEVZa96:b-caojhn) ([raw file](https://github.com/harness/harness-core/blob/3e6e8873eb224572266fd8c93c3bb8da140d4f02/960-ng-core-beans/src/main/java/io/harness/ng/core/envGroup/constant/EnvironmentGroupConstants.java#L7)):*
> ```Java
>   String ORG_PARAM_MESSAGE = "Organization Identifier for the Entity.";
>   String PROJECT_PARAM_MESSAGE = "Project Identifier for the Entity.";
>   String IDENTIFIER_PARAM_MESSAGE = "Environment Group Identifier Param Message";
> ```

Remove the "Param Message" from "Environment Group Identifier Param Message"

**Review by prashantsharma04 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/31423)* status: 4 file(s) to review. 1 change requests. 1. unresolved discussions. 0 of 2 approvals obtained. 

___
*[960-ng-core-beans/src/main/java/io/harness/ng/core/envGroup/constant/EnvironmentGroupConstants.java, line 7 at r1](https://reviewable.io/reviews/harness/harness-core/31423#-MyBAgM4Br8yF-EmEMso:-MyBWsEg-RA2AJyKGX8b:b-896fix) ([raw file](https://github.com/harness/harness-core/blob/3e6e8873eb224572266fd8c93c3bb8da140d4f02/960-ng-core-beans/src/main/java/io/harness/ng/core/envGroup/constant/EnvironmentGroupConstants.java#L7)):*
<details><summary><i>Previously, namanvermah (Naman Verma) wrote…</i></summary><blockquote>

Remove the "Param Message" from "Environment Group Identifier Param Message"
</blockquote></details>

Done.

File: 960-ng-core-beans/src/main/java/io/harness/ng/core/envGroup/dto/EnvironmentGroupResponse.java [hunk 1]
```diff
@@ -16,11 +19,11 @@
 @Data
 @NoArgsConstructor
 @FieldDefaults(level = AccessLevel.PRIVATE)
+@Schema(name = "EnvironmentGroup", description = "This is the UI view of Environment Group Entity defined in Harness")
```
Comment (vaibhav-si) [inline 960-ng-core-beans/src/main/java/io/harness/ng/core/envGroup/dto/EnvironmentGroupResponse.java:22]: No need to mention **UI View**, just **View** should be enough

**Review by prashantsharma04 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/31423)* status: 5 file(s) to review. 1 change requests. 3. unresolved discussions. 0 of 2 approvals obtained. 

___
*[960-ng-core-beans/src/main/java/io/harness/ng/core/envGroup/constant/EnvironmentGroupConstants.java, line 4 at r2](https://reviewable.io/reviews/harness/harness-core/31423#-My6SWBmI3gK7i_WZtAo-r2-4:-MyBuR5wBb9wXnjikooi:b-896fix) ([raw file](https://github.com/harness/harness-core/blob/c945dda85eaf5535c59f62125d6aa9d852acbc8f/960-ng-core-beans/src/main/java/io/harness/ng/core/envGroup/constant/EnvironmentGroupConstants.java#L4)):*
<details><summary><i>Previously, vaibhav-si (Vaibhav Singhal) wrote…</i></summary><blockquote>

reuse common constants like account, org, project etc. Have only env-specific constants in this file
</blockquote></details>

Done.
___
*[960-ng-core-beans/src/main/java/io/harness/ng/core/envGroup/dto/EnvironmentGroupResponse.java, line 22 at r2](https://reviewable.io/reviews/harness/harness-core/31423#-My6SWBmI3gK7i_WZtAq-r2-22:-MyBucnM41pFex-Joqz-:b-896fix) ([raw file](https://github.com/harness/harness-core/blob/c945dda85eaf5535c59f62125d6aa9d852acbc8f/960-ng-core-beans/src/main/java/io/harness/ng/core/envGroup/dto/EnvironmentGroupResponse.java#L22)):*
<details><summary><i>Previously, vaibhav-si (Vaibhav Singhal) wrote…</i></summary><blockquote>

No need to mention **UI View**, just **View** should be enough
</blockquote></details>

Done.

**Review by vaibhav-si — APPROVED** [review]
> Reviewed 1 of 4 files at r1, 4 of 4 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/31423)* status: :shipit: complete! ✓

**Pull request merged**